x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/edit_package_policy_page/utils/fix_apm_duration_vars.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DURATION_APM_SETTINGS_VARS } from '../../../../constants';
+
+// Fix duration vars, if it's a migrated setting, and it's a plain old number with no suffix, we should assume seconds
+export function fixApmDurationVars(vars: any) {
```

Comment:
could the type be more specific?

Reply:
hi, the type for `newVars` is `any` where we call the function, I can try to create a more specific type but not sure if it will work with other `vars` not APM specific

![image](https://user-images.githubusercontent.com/31922082/152154097-7044a86c-3a58-42a4-82be-02062a33b793.png)

Reply:
you could try with this: https://github.com/elastic/kibana/blob/2781a855e44b27fbea61e964fead4d7b7882e5a0/x-pack/plugins/fleet/common/types/models/package_policy.ts#L48